modules/router-store/src/router_store_module.ts

```diff
@@ -155,18 +165,31 @@ export function routerReducer<T = RouterStateSnapshot>(
   ],
 })
 export class StoreRouterConnectingModule {
```

Comment:
This would be a breaking change for existing users unless you add default providers to to NgModule metadata